src/components/QueryCondition.vue

```diff
@@ -91,9 +91,10 @@ export default Vue.extend( {
 				if ( selectedProperty === null ) {
 					this.$store.dispatch( 'unsetProperty', this.conditionIndex );
 					return;
+				} else if ( selectedProperty.datatype === 'wikibase-item' ) {
+					this.$store.dispatch( 'setSubclasses', { conditionIndex: this.conditionIndex, subclasses: true } );
```

Comment:
We do need to set this back to false when we select a different property, right?

Reply:
I added an else here

Reply:
:+1: